### PR TITLE
Make TBR requests acceptable on wearable (#1235)

### PR DESCRIPTION
**Pull request by quizzmaster**
I wrote some code to provide the ability to accept tbr requests on a wearable and dismiss the notifications on both the wearable and the phone on acceptance or invalidity of this request.
This implements the requested enhancement of #1235.

As my experience with android programming is very limited someone else with more experience should take a look at my implementation.

### CODE REVIEW

File: app/src/main/java/info/nightscout/androidaps/plugins/Wear/ActionStringHandler.java [hunk 1]
```diff
@@ -625,6 +650,35 @@ public synchronized static void handleConfirmation(String actionString) {
             doECarbs(carbs, starttime, duration);
         } else if ("dismissoverviewnotification".equals(act[0])) {
             MainApp.bus().post(new EventDismissNotification(SafeParse.stringToInt(act[1])));
+        } else if ("changeRequest".equals(act[0])) {
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/Wear/ActionStringHandler.java]: This is copy/paste from overview. Create a method in `LoopPlugin`

Comment (quizzmaster) [reply app/src/main/java/info/nightscout/androidaps/plugins/Wear/ActionStringHandler.java]: You are right. Moved the tbr accepting to a method in `LoopPlugin`. This method now uses its own event, so the GUI updating is also handled in one place now.

**Comment by MilosKozak** [comment]
ok now. who can test it?

**Comment by sethgagnon** [comment]
Here are the steps I performed to test this out:

1. Fetched the PR locally.
2. Built and generated full apk.
3. Uninstalled AAPS from watch and phone.
4. Installed AAPS on phone and then watch.
5. I did receive a notification on my phone for a TBR. I received on my watch as well.
6. I tapped on the notification I received on my watch and it opened to the full notification message.
7. There was an option to swipe to the left to then accept the TBR, which I did do. (I will say that the text here wasn't full screen. The box to scroll through the message was only about half the screen. You might be able to observe this in my screenshots.)
8. The notification disappeared on my watch and phone.
9. I went into the AAPS app on my phone and could see that the TBR had been accepted and was currently running (seeing the blue TBR line above IOB and COB).

Here are some screenshots for evidence.
phone_tbr_notification

watch_notification

opened_watch_notification_1

opened_watch_notification_2

accepted_running_tbr

**Comment by quizzmaster** [comment]
I used the existing Accept Activity which provides the scrolling text view. I didn't want to change the size of it, because this PR is only for making the tbr requests acceptable on a watch.

**Pull request merged**